spk/python3/Makefile

```diff
@@ -1,7 +1,7 @@
 SPK_NAME = python3
 SPK_SHORT_VERS = 3.6
 SPK_VERS = $(SPK_SHORT_VERS).8
-SPK_REV = 9
```

Comment:
Why the bump to 11? Since only 9 was published.

Reply:
It's only to keep track of the various attempts and have changelogs in case some options turned out to work better than the others.